src/themes/playful.css

```
@@ -334,14 +350,415 @@
+      --content-color: var(--wa-color-green-20);
+    }
+    &[variant='warning'] {
+      --background: var(--wa-color-yellow-80);
+      --content-color: var(--wa-color-yellow-20);
+    }
+    &[variant='danger'] {
+      --background: var(--wa-color-red-80);
+      --content-color: var(--wa-color-red-20);
+    }
+    &[variant='neutral'] {
+      --background: white;
+      --content-color: var(--wa-color-base-30);
+    }
+  }
+
+  .preview-container {
+    overflow: hidden;
+  }
+
+  .project-header {
+    --wa-color-text-normal: var(--wa-color-success-text-on-fill);
+  }
+
+  .hero-background {
+    position: absolute;
+    z-index: -1;
+    top: -18%;
+    left: 50%;
+    width: 250rem;
+    height: 250rem;
+    transform: translate(-50%, -50%);
+    flex-shrink: 0;
+    border-radius: 50%;
+    background: linear-gradient(180deg, var(--wa-color-green-90) 69.42%, var(--wa-color-green-80) 100%);
+  }
+
+  .hero-background::after {
+    background: url(/assets/images/kitchen-sink/playful/hero.png) 0 50% no-repeat;
+    content: '';
+    position: absolute;
+    width: 520px;
+    height: 600px;
+    transform: translateX(-50%);
+    left: 50%;
+    top: 185rem;
+  }
+
+  .hero {
+    padding-top: 19rem;
+    text-align: center;
+  }
+
+  .hero .hero-title {
+    color: var(--wa-color-success-text-on-fill);
+    font-size: var(--wa-font-size-4xl);
+  }
+
+  .hero .hero-cta {
+    font-size: var(--wa-font-size-l);
+  }
+
+  .product-card::part(footer) {
+    border-width: 0;
+  }
+
+  .product-card wa-button {
+    margin-inline-end: var(--wa-space-xs);
+  }
+
+  .blog wa-avatar {
+    --size: 2rem;
+  }
+
+  .message-composer wa-card {
+    --background: var(--wa-color-surface-raised);
+    --wa-border-width-s: calc(var(--wa-border-width-base) * 1rem);
+  }
+
+  .message-composer wa-card::part(header) {
+    --wa-color-neutral-border-subtle: var(--wa-color-base-70);
+    border-width: 0;
+    background-color: var(--wa-color-neutral-fill-subtle);
+  }
+
+  &:not(.wa-theme-premium-dark) .message-composer wa-card {
+    --wa-color-neutral-fill-subtle: var(--wa-color-base-90);
+  }
+
+  .message-composer wa-card::part(footer) {
+    padding-top: 0;
+  }
+
+  .product-detail .price {
+    font-weight: var(--wa-font-weight-heavy);
+  }
+
+  .support-table wa-card {
+    --background: var(--wa-color-surface-raised);
+    --border-radius: var(--wa-panel-corners);
+  }
+
+  .support-table th {
+    font-size: var(--wa-font-size-s);
+    font-weight: var(--wa-font-weight-heavy);
+    background-color: var(--wa-color-surface-lowered);
+  }
+
+  .support-table th:first-child {
+    border-radius: var(--wa-corners-l) 0 0 0;
+  }
+
+  .support-table th:last-child {
+    border-radius: 0 var(--wa-corners-l) 0 0;
+  }
+
+  .support-table tr:last-child td:first-child {
+    border-radius: 0 0 0 var(--wa-corners-l);
+  }
+
+  .support-table tr:last-child td:last-child {
+    border-radius: 0 0 var(--wa-corners-l) 0;
+  }
```

```suggestion
  .support-table th:first-child {
    border-radius: calc(var(--border-radius) - var(--border-width)) 0 0 0;
  }

  .support-table th:last-child {
    border-radius: 0 calc(var(--border-radius) - var(--border-width)) 0 0;
  }

  .support-table tr:last-child td:first-child {
    border-radius: 0 0 0 calc(var(--border-radius) - var(--border-width));
  }

  .support-table tr:last-child td:last-child {
    border-radius: 0 0 calc(var(--border-radius) - var(--border-width)) 0;
  }
```

Prevents peeking artifacts. This also allows us to change the card's border radius without needing to update these fellas.